src/ci/scripts/verify-line-endings.sh

```diff
@@ -4,21 +4,21 @@
 # We check both in rust-lang/rust and in a submodule to make sure both are
 # accurate. Submodules are checked out significantly later than the main
```

Comment:
src/tools/rust-installer is no longer a submodule (as of probably over a year ago). Can you update to check e.g. src/tools/cargo?

Reply:
Ok, I've done `src/tools/cargo/Cargo.lock` as it seems safe to assume that'll always exist.